### PR TITLE
Fixing internship listing

### PR DESCRIPTION
**Pull request by philspray**

### CODE REVIEW

File: app/views/content/teaching-internship-providers/_listing.html.erb [hunk 1]
```diff
@@ -60,3 +60,10 @@
   <li>Use the link to their website to find out further details and how to apply.</li>
   <li>Maximise the chances of your application succeeding with <a href="/explore-teaching-advisers">free one-to-one support from an explore teaching adviser</a>.</li>
 </ol>
+
+<h2> Apply for a teaching internship</h2>
```
Comment (gemmadallmandfe) [inline app/views/content/teaching-internship-providers/_listing.html.erb:64]: do we need this extra h2 @philspray ? (can remove retrospectively though!!)

**Review by StackedPancakez — APPROVED** [review]
Assuming CI passes, LGTM.

File: app/views/content/teaching-internship-providers/_listing.html.erb (hunk 1 above)
Comment (gemmadallmandfe) [reply app/views/content/teaching-internship-providers/_listing.html.erb]: Hmm just looked at the archive - maybe it needs an h3 intro eg previously we had an h3 of:

`Find a teaching internship in your region`

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - let's get this merged first thing

(@philspray let's add an h3 above the listings retrospectively - no rush for that!)

**Pull request merged**